HW 2/css/style.css

```diff
@@ -0,0 +1,129 @@
+body {
+    background-color: #f9f9f9;
+}
+a {
+    text-decoration: none;
+}
+@font-face {
+    font-family: 'OpenSans';
+    src: url(../fonts/OpenSans.ttf);
+}
+.wrapper {
+    margin: auto;
+    width: 1060px;
+    padding: 30px 47px;
+    background-color: #ffffff;
+    box-shadow: 0 0 10px rgba(0,0,0,0.5);
+    font-size: 13px;
+    font-family: 'OpenSans';
+}
+/*HEADER*/
+.logo+.header {
+    margin: 40px 0;
+}
+/*TOP MENU*/
+.top-menu {
+    background: linear-gradient(to top, #444343, #5e5e5e);
+    border: 1px solid #e1e1e1;
+    display: flex;
+    font-size: 15px;
+}
+.top-menu-item {
+    padding: 15px 30px;
+    color: #fff;
+    display: inherit;
+    border-left: 1px solid rgba(255,255,255, 0);
+    border-right: 1px solid rgba(255,255,255, 0);
+}
+.top-menu-item:hover {
+    background: linear-gradient(to top, #2e2e2e, #4a4a4a);
+    border-left: 1px solid rgba(255,255,255, 1);
+    border-right: 1px solid rgba(255,255,255, 1);
+}
+/*LEFT MENU*/
+.left-menu {
+    width: 240px;
+    float: left;
+}
+.left-menu-list {
+    border: 1px solid #5e5e5e;
+    border-top: none;
+}
+.left-menu-list:first-child {
+    border-top: 1px solid #5e5e5e;
+}
+.left-menu-item {
+    color: #656363;
+    display: block;
+    padding: 20px 50px;
```

Comment:
too big left-right padding, you can use this space for text. To allign text horizontally use `text-allign`